samples/test/writeClient.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it, before, after} = require('mocha');
+const uuid = require('uuid');
+const cp = require('child_process');
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const bigquery = new BigQuery();
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const GCLOUD_TESTS_PREFIX = 'nodejs_bqstorage_samples_test';
+
+const generateUuid = () =>
+  `${GCLOUD_TESTS_PREFIX}_${uuid.v4()}`.replace(/-/gi, '_');
+
+const datasetId = generateUuid();
+const tableId = generateUuid();
+
+describe('writeClient', () => {
+  let projectId;
+
+  before(async () => {
+    const schema = [
+      {name: 'bool_col', type: 'BOOLEAN'},
+      {name: 'bytes_col', type: 'BYTES'},
+      {name: 'float64_col', type: 'FLOAT'},
+      {name: 'int64_col', type: 'INTEGER'},
+      {name: 'string_col', type: 'STRING'},
+      {name: 'date_col', type: 'DATE'},
+      {name: 'datetime_col', type: 'DATETIME'},
+      {name: 'geography_col', type: 'GEOGRAPHY'},
+      {name: 'numeric_col', type: 'NUMERIC'},
+      {name: 'bignumeric_col', type: 'BIGNUMERIC'},
+      {name: 'time_col', type: 'TIME'},
+      {name: 'timestamp_col', type: 'TIMESTAMP'},
+      {name: 'int64_list', type: 'INTEGER', mode: 'REPEATED'},
+    ];
+
+    await deleteDatasets();
+
+    await bigquery.createDataset(datasetId);
+    const [table] = await bigquery
+      .dataset(datasetId)
+      .createTable(tableId, {schema});
+
+    projectId = table.metadata.tableReference.projectId;
+  });
+
+  after(async () => {
+    await bigquery.dataset(datasetId).delete({force: true}).catch(console.warn);
+  });
+
+  it('should append rows', async () => {
+    const output = execSync(
+      `node append_rows_proto2 ${projectId} ${datasetId} ${tableId}`
+    );
+    assert.match(output, /Stream created:/);
+    assert.match(output, /Row count: 13/);
+  });
```

Comment:
To check that the rows were properly committed, I'd like to see a query of the table here.

See: https://github.com/googleapis/python-bigquery-storage/blob/03966048f4a03f8f0a10009147076043bb9ea371/samples/snippets/append_rows_proto2_test.py#L64-L66

I think it has to be a query since I don't know if tabledata.list or the BQ Storage (Read) API can read from the vortex streaming buffer.

Reply:
Added